src/components/blog/post-nav.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { Search, ChevronDown, User, List } from 'react-feather'
+
+const PostNav = () => {
+  return (
+    <div className='container'>
+      <nav className='navbar navbar-expand-lg'>
+        <h3 className='navbar-brand'>More Posts</h3>
+        <ul className='navbar-nav ml-auto'>
+          <li className='nav-item'>
+            <a href='/blog' className='nav-link link-text'>
```

Comment:
Why is this set to the blog page?

Reply:
Good call. I will create this as its own component.